.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+
+jobs:
+  test:
+    docker:
+      - image: artsy/hokusai:0.4.0
+    working_directory: ~/metric
+    steps:
+      - add_ssh_keys
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Test
+          command: hokusai test
+  push:
```

Comment:
in Volt case we ended up doing push and deploy in one job as separate steps, this way after push you don't have to wait for another available container (which can take ~30 minutes if gravity or volt picked something up in the middle) http://artsy.github.io/blog/2018/01/19/upgrading-volt-to-circleci-two/

Reply:
Like so?

```yml
version: 2

jobs:
  test:
    docker:
      - image: artsy/hokusai:0.4.3
    working_directory: ~/volley
    steps:
      - add_ssh_keys
      - checkout
      - setup_remote_docker
      - run:
          name: Test
          command: hokusai test
  deploy:
    docker:
      - image: artsy/hokusai:0.4.3
    steps:
      - add_ssh_keys
      - checkout
      - setup_remote_docker
      - run:
          name: Push
          command: hokusai registry push --tag $CIRCLE_SHA1 --force --overwrite
      - run:
          name: Configure
          command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
      - run:
          name: Deploy
          command: hokusai staging deploy $CIRCLE_SHA1

workflows:
  version: 2
  default:
    jobs:
      - test
      - deploy:
          filters:
            branches:
              only: master
          requires:
            - test
```